tests/EvmAcceptanceTests/test/gas-transfer/GasTransferTest.ts

```diff
@@ -27,48 +33,56 @@ class TestCase {
     this.expectedContractAfterTransferIn = expectAfterTransfer;
     this.expectedContractAfterTransferOut = expectFinal;
   }
-};
+}
 
 describe("GasTransferTest", function () {
   const ONE_GWEI = BigNumber.from(1_000_000_000);
   const ONE_MWEI = BigNumber.from(1_000_000);
   const DEBUG = false;
   const testCases = [
     // Values less than 1MWei (10^(18-12)) won't show up because of rounding.
-    new TestCase( "0", BigNumber.from(ONE_GWEI), BigNumber.from(0), BigNumber.from(0), BigNumber.from(ONE_GWEI), BigNumber.from(ONE_GWEI), BigNumber.from(ONE_GWEI)),
- //   new TestCase( ONE_GWEI, BigNumber.from(0), BigNumber.from(0), BigNumber.from(0)),
- //   new TestCase( ONE_GWEI, ONE_GWEI, BigNumber.from(0), BigNumber.from(0)),
+    new TestCase(
+      "0",
+      BigNumber.from(ONE_GWEI),
+      BigNumber.from(0),
+      BigNumber.from(0),
+      BigNumber.from(ONE_GWEI),
+      BigNumber.from(ONE_GWEI),
+      BigNumber.from(ONE_GWEI)
+    )
+    //   new TestCase( ONE_GWEI, BigNumber.from(0), BigNumber.from(0), BigNumber.from(0)),
```

Comment:
dead code here

Reply:
I think this patch removes the dead code .. ?